### PR TITLE
Provide an example configuration (with space for more!)

### PR DESCRIPTION
**Pull request by Axarva**
## Description
Provides example configurations (#56), also fixes the infamous `informatiion` typo.

## Usage
Put files from the `examples/` directory **recursively** to your `~/.config/eww/` directory and launch the `bar` window.

### Showcase
I know, I know, showed this already but yeah

image

## Additional Notes
Added a line in the docs as well.

## Checklist

Please make sure you can check all the boxes that apply to this PR.

- [x] All widgets I've added are correctly documented.
- [x] The documentation in the `docs/content/main` directory has been adjusted to reflect my changes.
- [x] I used `cargo fmt` to automatically format all code before committing

### CODE REVIEW

**Comment by elkowar** [comment]
I think it would be nice if the different modules in the bar where sourced out into separate widget definitions, that are then all called from the main bar widget. 
As in, specify a `<def name="workspaces>` and include that in the `bar` definition, and so on - to show off how widgets can be nested and reused.

File: examples/scripts/getram [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (elkowar) [inline examples/scripts/getram:1]:
```suggestion
#!/usr/bin/env bash
```

File: examples/scripts/getvol [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (elkowar) [inline examples/scripts/getvol:1]:
```suggestion
#!/bin/sh
```

**Comment by elkowar** [comment]
also, it would be cool if you could add the screenshot to the Readme and the documentation too, maybe.

**Comment by Axarva** [comment]
> I think it would be nice if the different modules in the bar were sourced out into separate widget definitions, that are then all called from the main bar widget.
> As in, specify a `<def name="workspaces">` and include that in the `bar` definition, and so on - to show off how widgets can be nested and reused.

Did you mean to make multiple definitions for widgets and then include them all in the same `window`? 
As in different defs for everything and then include them all in the same window
I agree that would modularize stuff a lot but I think I tried that and it didn't work out due to widget having more than one child.
image

**Comment by Axarva** [comment]
> also, it would be cool if you could add the screenshot to the Readme and the documentation too, maybe.

Did it all, I wasn't somehow notified yesterday, so it took time, but yes, everything is done now lol

File: examples/scripts/getram (hunk 1 above)
Comment (Axarva) [reply examples/scripts/getram]: Just a habit lol

**Comment by elkowar** [comment]
> Did you mean to make multiple definitions for widgets and then include them all in the same window?

you can use user-defined widgets in other user-defined widgets. So you can create a `<time>` widget you can then use in a bar widget, which is then used in the bar window.

**Comment by Axarva** [comment]
Will try that and see how it works out :)

**Comment by Axarva** [comment]
@elkowar Does it now match what you had in mind?

**Comment by elkowar** [comment]
Yea, this is clean now! Thanks!

**Pull request merged**